src/main/java/com/elastic/support/diagnostics/commands/CollectSystemCalls.java

```diff
@@ -38,7 +38,7 @@ public void execute(DiagnosticContext context) {
             // Get the configurations for that platoform's sys calls.
             Map<String, String> sysCalls = osCmds.get("sys");
             processCalls(targetDir, sysCalls, sysCmd, pid);
-
+            logger.info( Constants.CONSOLE, "First set of system calls executed.");
```

Comment:
I added this log because this file do two different kind of request, and most of the time the second request fails and in the logs we only see `Unexpected error - bypassing some or all system calls.`.

The first group of request are the `sysstat` 99% of the time they works, but checking the logs we see only the error on the logs.
The request that fail are `javac`, `jps`... and this produce the error on the logs.

The try/catch cover the whole function :/ so this log it was added to add more info in the diagnostic output log file, so we know that at least the first set of system calls were executed.

I thought that this information was interesting, but I can remove it... and in the future we can split the function so we can have more specific logs and errors.

Reply:
That's reasonable. Let's just remove that random leading space then:

```suggestion
            logger.info(Constants.CONSOLE, "First set of system calls executed.");
```